addons/mail/static/src/js/form_renderer.js

```diff
@@ -97,6 +97,33 @@ FormRenderer.include({
     _hasChatter() {
         return !!this._chatterContainerTarget;
     },
+    /**
+     * Determine whether the chatter of the form renderer should display activities
```

Comment:
3rd person ("returns"), dot at the end, and limit to 80 char per line. (same for 2 other other methods below)